pkg/spec3/benchmark_serialization_test.go

```diff
@@ -22,7 +22,7 @@ var swaggerDiffOptions = []cmp.Option{cmp.Comparer(func(a spec.Ref, b spec.Ref)
 
 func TestOpenAPIV3RoundTrip(t *testing.T) {
 	var fuzzer *fuzz.Fuzzer
-	fuzzer = fuzz.NewWithSeed(1646791953)
+	fuzzer = fuzz.New()
```

Comment:
That's somewhat unrelated, but I think having the test be random is more likely to catch something.

Reply:
I'll revert that, I don't want the test to be flaky, but I'll run many instances of it before I do so.